html/semantics/links/downloading-resources/resources/inspect-header.py

```diff
@@ -1,18 +1,18 @@
 def main(request, response):
-    headers = [("Content-Type", "text/plain")]
-    command = request.GET.first("cmd").lower()
-    test_id = request.GET.first("id")
-    header = request.GET.first("header")
-    if command == "put":
-        request.server.stash.put(test_id, request.headers.get(header, ""))
+    headers = [(b"Content-Type", b"text/plain")]
+    command = request.GET.first(b"cmd").lower()
+    test_id = request.GET.first(b"id")
+    header = request.GET.first(b"header")
+    if command == b"put":
+        request.server.stash.put(test_id, request.headers.get(header, b""))
 
-    elif command == "get":
+    elif command == b"get":
         stashed_header = request.server.stash.take(test_id)
         if stashed_header is not None:
-            headers.append(("x-request-" + header, stashed_header))
+            headers.append((b"x-request-" + header, stashed_header))
 
     else:
-        response.set_error(400, "Bad Command")
-        return "ERROR: Bad Command!"
+        response.set_error(400, u"Bad Command")
+        return u"ERROR: Bad Command!"
```

Comment:
```suggestion
        return b"ERROR: Bad Command!"
```

Reply:
FWIW, the body can be either bytes or text. Looks like you prefer bytes?

Reply:
Two things led me to suggest bytes in a bunch of places. First, bytes string literals have to be ASCII-only, so if one sees `b` in front of a long chunk, one doesn't need to squint and look for anything non-ASCII to worry about, as it would have to be escaped. Second, less conversion, when bytes are what's going on the network.

But it's a preference I could be argued out of very easily, for example by noting that if we can eventually drop the `u` prefixes, then that variant will be shorter.